simln-lib/src/defined_activity.rs

```diff
@@ -36,8 +39,8 @@ impl fmt::Display for DefinedPaymentActivity {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(
             f,
-            "static payment of {} to {} every {}s",
-            self.amount, self.destination, self.wait
+            "[{:?}] static payment of {} to {} every {}s",
+            self.name, self.amount, self.destination, self.wait
```

Comment:
I don't think we need an option here, but if I'm wrong this needs cleanup:

This is going to print something ugly like `Some(name)`, rather inline a quick decomposing of the option.
`if let Some(name) = self.name { name } else { "" }`